From 815323dc7de73743f2e263975c40b1904a175c85 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 26 Mar 2023 13:27:20 +0300
Subject: [PATCH] [SW-0] gaudi3: put blocks hdr before reg map hdr

gaudi3_reg_map.h is using definitions from gaudi3_blocks_linux_driver.h,
so it must be included below it.

Change-Id: I38105ffb8677d93af06378d0060563fc8e54e6bb
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/common/common.c b/arch/gaudi3/common/common.c
index 71ca9d4..5625a51 100644
--- a/arch/gaudi3/common/common.c
+++ b/arch/gaudi3/common/common.c
@@ -5,8 +5,8 @@
 
 #include <preboot.h>
 #include <hl.h>
-#include <gaudi3_reg_map.h>
 #include <gaudi3_blocks_linux_driver.h>
+#include <gaudi3_reg_map.h>
 #include <mstr_if_rr_lbw_regs.h>
 #include <pcie_core_regs.h>
 #include <pcie_aux_regs.h>
